polar-core/src/partial/partial.rs

```diff
@@ -353,11 +385,70 @@ mod test {
     #[test]
     fn test_partial_isa_with_fields() -> TestResult {
         let p = Polar::new();
-        p.load_str("f(x: Post{id: 1});")?;
+        p.load_str(
+            r#"f(x: Post{id: 1});
+               f(x: Post{id: 1}) if x matches {id: 2};
+               f(x: Post{id: 1}) if x matches Post{id: 2};
```

Comment:
We are missing a case for a instance literal with more than one field. Will add.

Reply:
Line 394, below?

Reply:
That one fails, so we never actually check that the code worked since no binding is returned.